core/appeal/service.go

```diff
@@ -486,6 +493,367 @@ func validateAppealOnBehalf(a *domain.Appeal, policy *domain.Policy) error {
 	return nil
 }
 
+// Patch record
+func (s *Service) Patch(ctx context.Context, appeal *domain.Appeal, opts ...CreateAppealOption) error {
+	existingAppeal, err := s.GetByID(ctx, appeal.ID)
+	if err != nil {
+		return fmt.Errorf("error getting existing appeal: %w", err)
+	}
+
+	if existingAppeal.Status != domain.AppealStatusPending {
+		return ErrAppealStatusInvalid
+	}
+
+	createAppealOpts := &createAppealOptions{}
+	for _, opt := range opts {
+		opt(createAppealOpts)
+	}
+	isAdditionalAppealCreation := createAppealOpts.IsAdditionalAppeal
+
+	isAppealUpdated, err := validatePatchReq(appeal, existingAppeal)
+	if err != nil {
+		return err
+	}
+
+	if !isAppealUpdated {
+		// do not perform any operation since no field changed
+		return nil
```

Comment:
let's return error and non 2xx response in this case so user will be notified that there's no changes made?